BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -140,8 +144,9 @@ void CAN1_RX0_IRQHandler(void)
 {
     CAN_Receive(CAN1, CAN_FIFO0, &RxMessage);
 
+
     if ((RxMessage.StdId == 0x001)&&(RxMessage.IDE == CAN_ID_STD) && (RxMessage.DLC == 1)){
         // TODO: do stuff
         RxFlag = true;
     }
-}
+}
```

Comment:
Can you add a blank line at the end of the file? The compiler will get mad and give a warning if you don't

Reply:
got it!